modules/developer_manual/pages/app/advanced/code_signing.adoc

```diff
@@ -119,7 +119,7 @@ specify your private and the public key as well as the path to the
 application. A valid example looks like:
 
 ....
-./occ integrity:sign-app --privateKey=/Users/lukasreschke/contacts.key --certificate=/Users/lukasreschke/CA/contacts.crt --path=/Users/lukasreschke/Programming/contacts``
+./occ integrity:sign-app --privateKey=/Users/lukasreschke/contacts.key --certificate=/Users/lukasreschke/CA/contacts.crt --path=/Users/lukasreschke/Programming/contacts`
```

Comment:
we should exchange lukasreschke with eg pvince81 (or whomever) as he is now with nextcloud

Reply:
I changed it to `/home/user` as a generic POSIX-compliant user home directory. That way, in the future, when other users move on, the docs don't need to be updated yet again.